README.md

```diff
@@ -51,6 +58,8 @@ If you are on a `SNAPSHOT` version, you can force redownloads of modules/depende
  ./gradlew[.bat] clean build --parallel --refresh-dependencies
 ```
 
+Some options are in 'gradle.properties', and you add your own in order to improve gradle speed
```

Comment:
The limit between what should know gradle user and what cas should provide is not easy to define. I added this comment in order to make the user aware that the option "--parallel" has been added to the gradle.properties. 

The documentation was talking about it, I thought it was relevant to add some comment for that.